src/app/pages/component-viewer/_component-viewer-theme.scss

```diff
@@ -0,0 +1,57 @@
+@mixin component-viewer-theme($theme) {
+  $primary: map-get($theme, primary);
+  $accent: map-get($theme, accent);
+  $warn: map-get($theme, warn);
+  $background: map-get($theme, background);
+  $foreground: map-get($theme, foreground);
+
+  app-component-viewer {
+    color: md-color($foreground, secondary-text);
+
+    .docs-component-viewer-tabbed-content {
+      h1, h4 {
```

Comment:
Maybe you can add classes to these instead of targeting tag names? My concern with applying it to tag names is that these styles will affect things inside the code examples